roofit/multiprocess/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ ROOT_LINKER_LIBRARY(RooFitMultiProcess
         src/ProcessManager.cxx
         src/util.cxx
         src/Queue.cxx
+        src/FIFOQueue.cxx
+        src/PriorityQueue.cxx
```

Comment:
Both the new `FIFOQueue.h` and `PriorityQueue.h` header files are only included from `roofit/multiprocess/src`. That means you should put them in the `src` directory, not `res` (which is meant for headers that are also included by other build targets).

Or do you plan to include these headers from other libraries in the future? Anyway I think it's cleaner to not do so

Reply:
If you move them to `src`, you also don't need the `RooFit/MultiProcess` subdirectory structure anymore for these two header files.

Reply:
Makes sense, I will do this.